### ISSUE TITLE
Mark notifications as read/unread broken on second page and following

### ISSUE THREAD

**Issue by Waithamai** [diaspora/diaspora#7431]
On /notifications ("All notifications") marking entries as read/unread by clicking on the 'eye symbol' does noch work on the second page and following pages. (It does work in the dropdown and on the first page.) 
It's broken at least since v0.6.4.1.

**Comment by cmrd-senya** [comment]
Might that be a regression of mentions in comments UI changes?

**Comment by SuperTux88** [comment]
Probably a duplicate of #7250. Can you check if you can mark them as read/unread if you load them via dropdown?

**Comment by Waithamai** [comment]
Yep, in the dropdown everything works fine, also for old notifications that would be on the second page.

**Comment by SuperTux88** [comment]
And if you load notifcations from the second page in the dropdown, do they start to work on the page too? So scroll down in the dropdown and then try to mark as read on the page.

**Comment by SuperTux88** [comment]
You can also check the javascript console if you see the errors from #7250.

**Comment by Waithamai** [comment]
When I open /notifications page 2, and also open the dropdown and scroll to the according notification, then it works. When I close the dropdown or scroll to another notification it stops working again.

**Comment by SuperTux88** [comment]
I close this, because it is 99% a duplicate of #7250.

I don't understand what you mean with "When I close the dropdown or scroll to another notification it stops working again.". I can't reproduce that with the latest develop, but I can reproduce the behavior of #7250. After I loaded the notifications in the dropdown, they keep working (open/close the dropdown doesn't change anything), only after reloading the page (F5, or switching to another page) it's broken again, because the notifications aren't loaded in the dropdown anymore.

**Comment by denschub** [comment]
> After I loaded the notifications in the dropdown, they keep working (open/close the dropdown doesn't change anything), only after reloading the page (F5, or switching to another page) it's broken again, because the notifications aren't loaded in the dropdown anymore.

That's pretty much what she said, yeah. :)
